src/components/Network.js

```diff
@@ -149,7 +149,7 @@ class Network extends React.Component {
         if (isConnected) {
           return renderConnected(peer, conn);
         } else {
-          return renderDisconnected(peer);
+          return null;
```

Comment:
It seems useful if we still display peers that were recently disconnected, say less than 10 minutes ago.

Reply:
We currently don't have this information in the API. I will open an issue and, if we find it relevant, we improve the API and add it here.